src/main/java/duke/command/DeadlineCommand.java

```diff
@@ -0,0 +1,63 @@
+package duke.command;
+
+import duke.exception.DukeException;
+import duke.exception.MissingByDeadlineException;
+import duke.exception.MissingDetailsException;
+import duke.exception.UnknownDateTimeException;
+import duke.task.Deadline;
+import duke.task.TaskList;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+public class DeadlineCommand implements Command {
+    /**
+     * deadlineCommand Method creates Deadline Tasks.
+     *
+     * @param taskList      is the list of Tasks are saved and manipulated
+     * @param commandSuffix is the additional String that accompanies two-step commands
+     * @throws DukeException when multiple exceptions are caught (e.g. unfilled secondary input)
+     */
+    public static String run(TaskList taskList, String commandSuffix) throws DukeException {
+        try {
+            String[] byDeadline = commandSuffix.split(" /by ");
+
+            try {
+                //Initialising proTip; also used as a silent check for MissingByDeadlineException
+                String proTip = byDeadline[1];
+                boolean unknownDate = false;
+
+                try {
+                    try {
+                        DateTimeFormatter inputdTFormat = DateTimeFormatter.ofPattern("d/M/yyyy HHmm");
+                        LocalDateTime outputDT = LocalDateTime.parse(byDeadline[1], inputdTFormat);
```

Comment:
outputDT or outputdT? Can be consistent :)
Also this is linked to my comment on the 'dt' naming

Reply:
Modified the input/output DT variables for the `DateTimeFormatter` objects, for a bit more consistency, but still complying to checkstyle. Leaving `outputDT` as is for a direct contrast that this is a `LocalDateTime` object instead.